app/presenters/filters_presenter.rb

```diff
@@ -20,10 +20,33 @@ def summary_items
   end
 
   def reset_url
-    "#"
+    return nil unless any_filters?
```

Comment:
Just to check, given this previously always returned a string is the view logic ok with a nil.

oh and nit the nil is superfluous:

```suggestion
    return unless any_filters?
```

Reply:
I stylistically like to explicitly return nil in query methods (that are intended to return data) and only use the short form in command methods that never return data.

But happy to leave it out if that's house style?

Reply:
Sure I can understand that. It's the more common approach for us but not a written rule so not a blocker if you think it improves comprehension of the code.